packages/core/src/plugins/amqp.ts

```diff
@@ -288,13 +289,20 @@ export function attach(this: Microfleet, opts: any = {}) {
       return true
     },
 
+    requestCount() {
```

Comment:
I suggest to call the method `getRequestCount()`, I prefer verbs in method naming, because they generally *do* something.